src/components/NavBar.jsx

```diff
@@ -0,0 +1,11 @@
+const NavBar = () => {
+    return (
+        <nav className="nav">
+            <ul>
+                <li>Future NavBar Here</li>
+            </ul>
+        </nav>
+    )
+}
+
+export default NavBar
```

Comment:
Usually we'd want whole, completed features in a PR - try to avoid WIP stuff if you can, the PR should be a clean, ready to go, fairly polished version of a feature

Reply:
oh great thanks for the reminder, I will focus on the current feature only from now on and avoid adding future feature placeholders, thanks so much for the helpful feedback 😄 🙌